azure/template.json

```
@@ -201,6 +202,47 @@
             "dependsOn": [
                 "[variables('resourceGroupName')]"
             ]
+        },
+        {
+            "apiVersion": "2019-05-10",
+            "name": "[concat(variables('keyVaultName'), '-', parameters('utcValue'))]",
+            "type": "Microsoft.Resources/deployments",
+            "properties": {
+                "mode": "Incremental",
+                "templateLink": {
+                    "uri": "[concat(variables('deploymentUrlBase'),'keyvault.json')]",
+                    "contentVersion": "1.0.0.0"
+                },
+                "parameters": {
+                    "keyVaultName": {
+                        "value": "[variables('keyVaultName')]"
+                    },
+                    "enabledForTemplateDeployment": {
+                        "value": true
+                    },
+                    "enableFirewall": {
+                        "value": true
+                    },
+                    "enableRbacAuthorization": {
+                        "Value": true
+                    },
+                    "subnetResourceIdList": {
+                        "value": "[parameters('sharedBackEndSubnetResourceId')]"
+                    },
+                    "allowTrustedMicrosoftServices": {
+                        "value": true
```

```suggestion
                        "value": false
```
Let's attempt with false initially